app/Http/Controllers/ResetController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ResetRequest;
+use App\Http\Requests\UpdatePasswordRequest;
+use App\Mail\SendMailreset;
+use App\Models\Reset;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Str;
+
+class ResetController extends Controller
+{
+    public function sendEmail(ResetRequest $request)
+    {
+        $email = $request->email;
+        $user = User::where('email', $email);
+        $reset = Reset::where('email', $email)->first();
```

Comment:
its better to load user by id, the one you should save in the token model and have a field for.
the reset model is also better to load by token, more correct to avoid some specific issues.